paywall/src/paywall-script/index.ts

```diff
@@ -163,6 +181,16 @@ export class Paywall {
     this.checkKeysAndLock()
   }
 
+  handleMethodCallEvent = async ({ method, params, id }: MethodCall) => {
+    // TODO: handle getting provider
```

Comment:
Do you want to do that in a later PR?
Let's have a discussion about this tomorrow but I think in practice we will need to use the injected provider by default anyway :/